lib/standards/aria-roles.js

```diff
@@ -1,4 +1,5 @@
 // Source: https://www.w3.org/TR/wai-aria-1.1/#roles
+// Source for abstract roles: https://www.w3.org/TR/wai-aria/#abstract_roles and https://www.w3.org/WAI/ARIA/1.2/class-diagram/rdf_model.svg
```

Comment:
According to https://www.w3.org/WAI/ARIA/1.2/class-diagram/rdf_model.svg `tabpanel` is a section, which is under structure.

Reply:
Don't reference the svg instead we make spec difference for tabpanel to not be `widget`

Reply:
Removed svg as comment reference 2d185f65